pkg/email/templates/cedar_roles_changed.gohtml

```diff
@@ -6,6 +6,7 @@
 	<li>Customer: EASi</li>
 	<li>System: {{.SystemName}}</li>
 	<li>API: CEDAR Core</li>
+	<li>Requester {{.RequesterFullName}}</li>
```

Comment:
Small note - this is missing a ":", so this line in the email appears as "Requester Requester Jones", instead of "Requester: Requester Jones".

Reply:
b00b2c126

Reply:
Thanks for the catch!